aesara/tensor/basic.py

```diff
@@ -1829,12 +1567,8 @@ def perform(self, node, inp, out_):
 default = Default()
 
 
-##########################
-# View Operations
-##########################
-
-
-def extract_constant(x, elemwise=True, only_process_constants=False):
+# TODO: Remove this function.
```

Comment:
If this function is useful, and there's not much alternatives to it, we can add a `raises` keyword argument `get_scalar_constant_value` and make this function call inside `get_scalar_constant_value`. Something like:

```python
def extract_constant(x, only_process_constants=False, raises=True):
    if not raises:
      if isinstance(x, aes.ScalarVariable) or isinstance(
          x, aes.sharedvar.ScalarSharedVariable
      ):
          if x.owner and isinstance(x.owner.op, ScalarFromTensor):
              x = x.owner.inputs[0]
          else:
              x = tensor_from_scalar(x)
      return x
    raise NotScalarConstantError()
```